index.js

```diff
@@ -268,6 +274,10 @@ class Raspi extends EventEmitter {
           analogChannel: {
             enumerable: true,
             value: 127
+          },
+          [isHardwarePwm]: {
```

Comment:
Nit: so I know this is kinda confusing, but the `pins` object and the `instances` object do have separate purposes. The `pins` object is the one that is exposed for use by J5, and is fullfilling the contract spelled out in the [IO Plugin specification](https://github.com/rwaldron/io-plugins#minimum-plugin-class-requirements). The `instances` object, however, is used for tracking internal state specific to Raspi IO. As such, this property should be on the `instances` object (probably around line 235 or so), not on the `pins` object. As an added bonus, by moving it to the `instances` object, it doesn't have to be a symbol anymore, it can be a normal property.